### DISCUSSION TITLE
11월 3주차 회고록

### DISCUSSION THREAD

**Discussion by KwonNaeHyeon** [BDD-CLUB/challenge-every-week-posting#245] [5. 회고]
[공부와 수능](https://lateral-value-9a9.notion.site/11-387d7b801c534d3b8c734b5d4c2e5e15?pvs=4)
안녕하세요. 부쩍 추워진 날씨 때문인지 기력이 쇠하고 지치기 쉬운 11월입니다. 다들 감기 조심하시고 몸도 마음도 건강할 수 있길 기원하겠습니다. 감사합니다.

**Comment by shkisme** [comment]
동생께서 무사히 수능을 끝내서 다행입니다...휴...😮‍💨
